packages/serverpod/lib/src/server/serverpod.dart

```diff
@@ -212,13 +212,32 @@ class Serverpod {
   /// started.
   late final ClusterManager cluster;
 
+  /// Http headers used by all API responses. Defaults to allowing any
+  /// cross origin resource sharing (CORS).
+  final Map<String, dynamic> httpResponseHeaders;
+
+  /// Http headers used by OPTIONS responses.
+  final Map<String, dynamic> httpOptionsResponseHeaders;
+
+  static const _defaultHttpResponseHeaders = {
+    'Access-Control-Allow-Origin': 'http://localhost:8082',
```

Comment:
This will clearly break things when not running on localhost (e.g. deploying to the cloud). It needs to be set to `'*'`.